### PR TITLE
completed react-carousel-component challenge

### PR DESCRIPTION
**Pull request by Stran1025**
react carousel

### CODE REVIEW

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Where are the dots though?

**Comment by Stran1025** [comment]
> Where are the dots though?

Pikachu ate them, but I found them!
react carousel

File: react-carousel-component/src/carousel.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          })}
+          <i className='fas fa-angle-right fa-2xl right' onClick={this.handleRight}></i>
+        </div>
+        <div className='dot-holder'>
+          {this.props.input.map((img, index) => {
+            let active = '';
+            if (this.state.currentImg === index) {
+              active = 'active';
+            }
+            return <div key={index} className={'dot ' + active}></div>;
```
Comment (thebearingedge) [inline react-carousel-component/src/carousel.jsx:63]: Please check the instructions for how the dots should work

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: react-carousel-component/src/carousel.jsx (hunk 1 above)
Comment (Stran1025) [reply react-carousel-component/src/carousel.jsx]: react carousel

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**